### PR TITLE
Added BepInEx for V Rising support.

### PR DESCRIPTION
**Pull request by ricin**

### CODE REVIEW

**Comment by ich777** [comment]
@ricin Thank you for the PR, really much appreciated!

I will go through the PR today.
First I don't wanted to implement BepInEx (a Forum member of the Unraid forums already requested this [here](https://forums.unraid.net/topic/79530-support-ich777-gameserver-dockers/?do=findComment&comment=1134006)) but I told him that I would like to wait until the native Linux client from V-Rising is released.

Will let you know when I tested this myself.

**Comment by ich777** [comment]
Please let me know what you think of those changes that I've made so that I can maybe merge the PR and update the container if you are happy with it.

File: scripts/start-server.sh [hunk 1]
```diff
@@ -53,17 +53,13 @@ else
         +quit
     fi
 fi
-BEPINEX_VR_TS_API_URL=https://thunderstore.io/c/v-rising/api/v1/package/b86fcaaf-297a-45c8-82a0-fcbd7806fdc4/
-BEPINEX_VR_TS_URL=https://v-rising.thunderstore.io/package/BepInEx/BepInExPack_V_Rising/
-BEPINEX_VR_TS_JSON=${SERVER_DIR}/BepInExPack_V_Rising.json
 
 if [ "${ENABLE_BEPINEX}" == "true" ]; then
-    export  WINEDLLOVERRIDES="winhttp=n,b"
```
Comment (ricin) [inline scripts/start-server.sh]: Without this BepInEx won't be loaded. It's needed for wine to pick up the winhttp.dll in the V Rising folder and load BepInEx.

Comment (ich777) [reply scripts/start-server.sh]: Thank you for the hint, adding the line again.
Will update the container shortly.

**Pull request merged**